Backend/BikesRentalServer.DataAccess/Repositories/Abstract/IReservationsRepository.cs

```diff
@@ -1,9 +1,11 @@
 ﻿using BikesRentalServer.Models;
+using System.Collections.Generic;
 
 namespace BikesRentalServer.DataAccess.Repositories.Abstract
 {
     public interface IReservationsRepository : IRepository<Reservation>
     {
         Reservation GetActiveReservation(string bikeId);
+        List<Reservation> GetActiveReservationsForUser(string userId);

```

Comment:
Dla porządku warto by albo tę metodę nazwać `GetActiveReservations` (a że dla usera to wynika z tego, że podajemy userId, ew. dać xml docs z opisem), albo tę powyżej zamienić na `GetActiveReservationForBike` (lub jakieś wariacje np. `GetActiveBikeReservation`, `GetActiveUserReservations`)

Reply:
Pewnie, zaraz podmienie ;)